File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.java
class Solution {
    public String solution(String my_string, int s, int e) {
        String answer = my_string.substring(0,s);
        StringBuilder sb = new StringBuilder();
        sb.append(my_string.substring(s,e + 1)).reverse();
        return answer + sb.toString() + my_string.substring(e + 1,my_string.length());
    }
}

File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.py
def solution(my_string, s, e):
    answer = list(my_string)
    answer[s:e + 1] = reversed(answer[s:e + 1])
    return ''.join(answer)

    # return my_string[:s] + my_string[s:e + 1][::-1] + my_string[e + 1:]